mailbox/api/src/main/java/org/apache/james/mailbox/model/MessageResult.java

```diff
@@ -244,6 +247,13 @@
     Headers getHeaders() throws MailboxException;
     
     /**
+     * Return the list of attachments
+     * 
+     * @return a read only list of attachments
```

Comment:
Immutable is the default, we should not need comment about that, don't you think ?